didx-xyz/yoma-mobile
# YOMA-604 - feat(Education form): wire up orgs and skills

## Scope [YOMA-604](https://yomaworld.atlassian.net/browse/YOMA-604)

![](https://media.giphy.com/media/hjXEu11TDY5mE/giphy.gif)

## Work Done
added selector for orgs and skills dropdowns.
Fixed some build issues.

## Steps to Test
run the app - open the Education form - see that it's pulling real org and skill data.
